addons/mail/static/src/chatter/web/chatter_patch.js

```diff
@@ -16,22 +21,143 @@ Chatter.components = {
     FollowerList,
 };
 
-Chatter.props = [...Chatter.props, "has_activities?"];
+Chatter.props = [...Chatter.props, "has_activities?", "hasParentReloadOnFollowersUpdate?"];
 
-Chatter.defaultProps = { ...Chatter.defaultProps, has_activities: true };
+Chatter.defaultProps = {
```

Comment:
Similarly, shouldn't it be object.assign?

Reply:
Both syntax works. If there is a prefer, I can change it.

Reply:
It's the same idea as the state, if it's not changed in place, it might fail to work with other overrides or local code storing references.